Set binary name to ezquake instead of ezquake-OpenBSD-<arch>

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -885,6 +885,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     string(TOLOWER "ezQuake" EXECUTABLE_NAME)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
     string(TOLOWER "ezquake" EXECUTABLE_NAME)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+    string(TOLOWER "ezquake" EXECUTABLE_NAME)
 else()
     string(TOLOWER "ezquake-${CMAKE_SYSTEM_NAME}-${CMAKE_SYSTEM_PROCESSOR}" EXECUTABLE_NAME)
 endif()
